mppm/virtual-players/virtual-players-layout.md

```diff
@@ -0,0 +1,34 @@
+---
+id: virtual-players-layout
+title: Change the layout of a Virtual Player
+description: Learn how to change the layout of a Vrtual Player window.
+---
+
+You can display and use the following parts of the [Unity Interface](https://docs.unity3d.com/Manual/UsingTheEditor.html) in a Virtual Player:
+- [Console](https://docs.unity3d.com/Manual/Console.html)
+- [Game](https://docs.unity3d.com/Manual/GameView.html)
+- [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html)
+- [Inspector](https://docs.unity3d.com/Manual/UsingTheInspector.html)
+- [Scene](https://docs.unity3d.com/Manual/UsingTheSceneView.html)
```

Comment:
Can we also add here that the PlayMode Tools window is available also both in and out of playmode if they have the netcode for entities package installed and then we can link to the documentation we put about that window